src/arviz_plots/plots/__init__.py

```diff
@@ -2,6 +2,7 @@
 
 from .autocorr_plot import plot_autocorr
 from .bf_plot import plot_bf
+from .combine import combine_plots
```

Comment:
Is the name `combine_plots` intentional instead of `plot_*`? It makes sense to make the name really different, jsut checking. 
Are we happy with `combine` or do we prefer alternatives like `layout`, `grid`, `arrange`?

Reply:
I am ok with `combine_plots` but I wouldn't oppose something like `plot_layout`, `plot_grid` (this one might be confusing with the .grid method of plotcollection though) or `plot_combined` to keep the `verb_name` pattern

Reply:
No strong preference either.